ref.gemspec

```diff
@@ -5,11 +5,11 @@ require 'ref/version'
 Gem::Specification.new do |s|
   s.name        = 'ref'
   s.version     = Ref::VERSION
-  s.authors     = ['Brian Durand']
+  s.authors     = ['Brian Durand and concurrent-ruby team']
   s.email       = ['bbdurand@gmail.com']
```

Comment:
I did something similar in the [gemspec on this PR](https://github.com/ruby-concurrency/concurrent-ruby/pull/268/files). Should we standardize? Perhaps something like this?

```
s.authors   = ["Brian Durand", 'The Ruby Concurrency Team']
s.email     = ['bbdurand@gmail.com', 'concurrent-ruby@googlegroups.com']
```